R/summarise_country.R

```diff
@@ -0,0 +1,163 @@
+summarise_sheet_indicator_list <- function(billion) {
+  billion <- rlang::arg_match(billion)
```

Comment:
This use of `arg_match` won't work because the function doesn't have default values. You need to specify possible values if defaults aren't there in the function definition. Probably better to do check in another function anyway.

Reply:
Fair. I had put it as a skeleton to be completed later (as you will see in other parts of the code).